test/test_sparse.py

```diff
@@ -706,6 +710,17 @@ def test_is_sparse(self):
         self.assertTrue(x.is_sparse)
         self.assertTrue(torch.autograd.Variable(x).is_sparse)
 
+    def test_resize_as(self):
+        from torch.autograd import Variable
+        x = self.SparseTensor()
+        v = Variable(x)
+
+        def empty_like(t):
```

Comment:
does torch.empty_like not work for you?

Reply:
I'm going to add a few more assertions to the test, but the goal was to test `resize_as_` directly, not through `empty_like` which may not call it.